src/components/sync-connected-sites.tsx

```diff
@@ -213,7 +213,7 @@ const SyncConnectedSitesSection = ( {
 							key={ connectedSite.id }
 							className="flex items-center gap-2 min-h-14 border-b border-a8c-gray-0 px-8"
 						>
-							<div className="flex items-left min-w-20 me-6 shrink-0">
+							<div className="flex items-left min-w-[100px] me-4 shrink-0">
```

Comment:
```suggestion
							<div className="flex items-left min-w-28 me-4 shrink-0">
```
I think we should try to stick to Tailwind presets when possible. In this case, this suggestion seems to work fine.